e2e/editor-tabs.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(
+    '/learn/2022/responsive-web-design/build-a-survey-form-project/build-a-survey-form'
+  );
+});
+test.afterEach(async ({ page }) => {
+  await page.close();
+});
+
+test('should toggle editor visibility based on aria-expanded attribute', async ({
+  page
+}) => {
+  // Find the first button that is not expanded (aria-expanded="false")
+  const notExpandedButton = await page.$('button[aria-expanded="false"]');
```

Comment:
A matter of curiosity, why are we using `$` and `$$` instead of more human-readable methods?

Reply:
I am not sure how to make it more human-readable... Can you show me in this example and I'll do the rest.
thank you!

Reply:
`page.locator()` would work, though it would be better to add a playwright test id to the element and query that.

Reply:
I tried adding an id to the button and do 
```
const notExpandedButton = page.locator('#editor-tab-button[aria-expanded="false"]');
```
but it seems unable to locate the button and I got "Test timeout of 15000ms exceeded."

I also tried adding a `data-testid="directions"` to the button and I did
```
const button = page.getByTestId('directions'); 
```
but if there are multiple buttons the test fails because it locates more than one element.

I am not sure what else I can do now...

Reply:
Hi Naomi, I updated the code using page.locator(), could you please take a look? @naomi-lgbt